src/app/Header.scss

```diff
@@ -158,6 +158,10 @@ header {
 
       &.active {
         box-shadow: inset 0 -1px 0 $primary-color;
+
+        i {
```

Comment:
Please style `.icon` instead of html `i` tag. It helps to make it more clear and avoid css conflicts.